testing/test_terminal.py

```diff
@@ -964,3 +964,58 @@ def test_no_trailing_whitespace_after_inifile_word(testdir):
     testdir.makeini('[pytest]')
     result = testdir.runpytest('')
     assert 'inifile: tox.ini\n' in result.stdout.str()
+
+
+class TestProgress:
+
+    @pytest.fixture
+    def many_tests_file(self, testdir):
+        testdir.makepyfile(
+            test_bar="""
+                import pytest
+                @pytest.mark.parametrize('i', range(10))
+                def test_bar(i): pass
+            """,
+            test_foo="""
+                import pytest
+                @pytest.mark.parametrize('i', range(5))
+                def test_foo(i): pass
+            """,
+            test_foobar="""
+                import pytest
+                @pytest.mark.parametrize('i', range(5))
+                def test_foobar(i): pass
+            """,
+        )
+
+    def test_normal(self, many_tests_file, testdir):
+        output = testdir.runpytest()
+        output.stdout.re_match_lines([
+            r'test_bar.py \.\.\.\.\.\.\.\.\.\. \s+ \[ 50%\]',
```

Comment:
Might want to do `\.{10}` here and in other tests.

Reply:
Definitely 👍